packages/dai-plugin-mcd/test/ManagedCdp.spec.js

```diff
@@ -58,13 +58,29 @@ test('getLiquidationPrice and getCollateralizationRatio returns infinity with 0
   expect(collateralization).toBe(Infinity);
 });
 
-async function expectValues(cdp, { collateral, debt, myGem, myDai }) {
-  cdp.reset();
+async function expectValues(
+  cdp,
+  {
+    reset,
+    collateral,
+    debt,
+    myGem,
+    myDai,
+    collateralValue,
+    ratio,
+    isSafe,
+    daiAvailable
+  }
+) {
+  if (reset) {
+    cdp.reset();
+    await cdp.prefetch();
+  }
```

Comment:
`expectValues` probably shouldn't also reset the cdp as it's a bit unexpected - I was a bit confused at first as to why we were expecting `reset` to be true on a cdp :) 

What do you think about pulling resetting out into a separate function?